gems/aws-sdk-apigateway/features/smoke.feature

```diff
@@ -0,0 +1,27 @@
+# WARNING ABOUT GENERATED CODE
+#
+# This file is generated. See the contributing guide for more information:
+# https://github.com/aws/aws-sdk-ruby/blob/master/CONTRIBUTING.md
+#
+# WARNING ABOUT GENERATED CODE
+
+
+Feature: Smoke tests for APIGateway
+
+Background:
+  Given I create a client in region 'us-west-2'
+
+  @apigateway @smoke
```

Comment:
This is fine, but probably smoke_tags can be per file base instead of per test case?